plugins/woocommerce-blocks/assets/js/blocks/product-gallery/frontend.tsx

```diff
@@ -205,6 +217,47 @@ const productGallery = {
 				'wc-block-product-gallery-dialog-open'
 			);
 		},
+		onTouchStart: ( event: TouchEvent ) => {
+			const context = getContext();
+			context.touchStartX = event.touches[ 0 ].clientX;
+			context.touchCurrentX = event.touches[ 0 ].clientX;
```

Comment:
Super nitpick, you can leave it as-is 🤷 up to you

```suggestion
            const { clientX } =  event.touches[ 0 ];
			context.touchStartX = clientX;
			context.touchCurrentX = clientX;
```

Reply:
https://github.com/woocommerce/woocommerce/pull/55010/commits/2a9b9a55a3457f4dc99443993d51442cece4ebc9